packages/ucp/src/util/Vocabularies.ts

```diff
@@ -0,0 +1,135 @@
+import { DataFactory } from 'n3';
+import type { NamedNode } from '@rdfjs/types';
+
+// shameless copy of the Community Solid Server Vocabularies
```

Comment:
Why not import those?

Reply:
Because then you have to add the CSS as a dependency to this library. 
I've asked Joachim a couple of times to make this vocabulary methods a stand alone package and he refused, so this is my solution for that.

Reply:
Hm, alright. I wouldn't have a big issue with temporarily depending on CSS, but let's leave it as is then.

I am also pushing Joachim to split the generic utility code off from the main CSS package, and I had the feeling that we will someday get there. We'll see.